src/core/dev_settings.py

```diff
@@ -10,4 +10,17 @@
 URL_CONFIG = 'path'  # path or domain
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-MIDDLEWARE_CLASSES = ('utils.middleware.TimeMonitoring',)
+MIDDLEWARE_CLASSES = (
+    'utils.middleware.TimeMonitoring',
+    'debug_toolbar.middleware.DebugToolbarMiddleware'
+)
+INSTALLED_APPS = ['debug_toolbar']
+
```

Comment:
spacing looks inconsistent

Reply:
Not sure why you mean here clarification?

Reply:
There is one space between constants but two between a constant and a function